text/0000-general-fund.md

```diff
@@ -0,0 +1,110 @@
+
+Everyone is welcome to weigh in on the merits of a funding request in the forum thread and during the public governance meeting where the funding request is finalized. The core team will strive to ensure there's a community consensus for each request, however the final decision rests with the core team. Other Grin teams who have been allocated budgets can spend funds at their own discretion, but are strongly encouraged to consider the guidelines in this RFC during their decision making process.
+
+## Tips on effective proposals
+
+* You proposal is likely to be better received if you're already known to the Grin community or the wider cryptocurrency community. Even more so if you've already contributed to Grin in some meaningful way.
+* It might be better to keep your initial funding request small and focused, particularly if your idea is something new and radical, or if you're unknown to the Grin community.
+* Be specific, professional and thorough with your proposal.
+* Be respectful and professional in your response to any feedback and criticism on the proposal thread 
+
+# Payout Process
+
+* At the time of payout, an amount of Bitcoin equivalent to the funding amount is paid out to an address provided by the funding requestor only.
+* The payout address should be provided to the core team by the requestor via a secure, verifiable channel (with cryptographic proof of identity if deemed necessary).
+* The exchange rate between BTC and the target currency is determined by the exchange rate at the time of the funding transaction, without exception.
+* To minimize the effects of currency fluctuations on accounting transparency, all payments should be performed at pre-arranged "signing parties" in which mu-sig holders sample the current exchange rate. The transaction should be created, signed and broadcast as quickly as possible, with the transaction and rate reported back to the core team for tracking.
+* Mu-sig holders will verify the address provided by the funding requestor several times during transaction creation and broadcasting. However, the core team is not responsible for errors or incorrect addresses provided by the requestor.
+
+## Transparency and processes
+
+The result of all funding decisions will be published in the meeting notes found in the [Grin Project Management](https://github.com/mimblewimble/grin-pm) repository.
+
+The core team is obliged to publish a detailed spending log of all transactions made in and out of the fund, as well as quarterly spending reports summarizing all income and spending. These reports will also be made available in the Grin PM Repository.
+
+## Unresolved questions
+
+* This RFC is put together on the understanding that the fundamental question of 'who appoints the core team' is still outstanding. This point should be discussed separately and this RFC assumes there is an acceptable answer in place.
+
+# Future possibilities
```

Comment:
1. Future possibilities section is empty. Either add some future possibilities (for example a community funding process where members make donations or show support for proposals?), or   remove the section altogether. 

1. References section is missing. Is that deliberate? You might want to add references to grin-pm/financials and decision log in grin-pm and an example funding request etc. Just a thought, if you don't think it's necessary then leave it all out I guess.